creativecloud/features/interactive-elements/interactive-elements.css

```diff
@@ -280,67 +247,38 @@ div.genfill-prompt>p:first-child {
 }
 
 @media (min-width: 600px) and (max-width: 1199px) {
-  /* Firefly interactive components position */
-  .firefly-prompt {
-    top: 32px;
-    margin-left: 32px;
-    margin-right: 32px;
-  }
-
-  .genfill-promptbar {
-    top: 40px;
-    margin-left: 32px;
-    height: 40px;
-  }
-
-  .firefly-selectortray {
-    top: 0;
-    margin-left: 42px;
-    margin-right: 42px;
-    width: 80%;
-  }
-
-  /* Prompt Bar */
   .promptbar {
     height: 40px;
-    padding: 7px 7px 7px 32px;
-    width: 465px;
+    padding-inline-start: 32px;
+    padding-inline-end: 7px;
+    width: 464px;
   }
 
   .prompt-text {
-    min-width: calc(100% - 126px);
+    width: 325px;
     font-size: var(--type-body-l-size);
     line-height: var(--type-body-l-lh);
```

Comment:
Can we add body-l directly?

Reply:
prompt text is not same for all breakpoints hence if we if we use class name we'll have to handle that in js to update the css class. Hence defining in css file using variable